### PR TITLE
Capstone_readme_content

### PR DESCRIPTION
**Pull request by BenjaminMatembe**
On this feature using Javascript, HTML and CSS I:

- Added the home page and about page as per the design guidelines from [Cindy Shin in Behance](https://www.behance.net/adagio07).

 > The two pages are responsive on two media queries and these are:

- [Mobile: up to 768px wide] 
-    [Desktop: 768px or wider] 

> The mobile version has a hamberger menu that opens for the side navbar and the menu that opens closes through a click on the X button.

- I also added a dynamic section (speakers section) using javascript array of objects, backticks and a for loop plus the DOM.

- [ ] here is the link to the description of the project [loom video](https://www.loom.com/share/af978cc61bdd4706b6da3917ae768fb6)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            <li><a href="about.html">About</a></li>
+            <li><a href="">Program</a></li>
+            <li><a href="">Join</a></li>
+            <li><a href="">Sponsor</a></li>
+            <li><a href="">News</a></li>
+        </ul>
+        
+        </main>
+        
+    </section>
```
Comment (Qoosim) [inline about.html:24]: - [ ] It is expected that there should a link from from about page on mobile to home page and vice-versa. At this moment, none of the links redirects to the either of the home or about from both pages on mobile.

[mobile-menu-behave.webm](https://user-images.githubusercontent.com/26597432/233061780-0347a8f8-44a0-4a54-acf1-2f6f3b1a53a7.webm)

File: about.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            <ul>
+                <li><a href="" class="aboutDesktop">About</a></li>
+                <li><a href="" class="aboutLi">Program</a></li>
+                <li><a href="" class="aboutLi">Join</a></li>
+                <li><a href="" class="aboutLi">Sponsor</a></li>
+                <li><a href="" class="aboutLi">News</a></li>
+                <li ><a href="" class="border">cc campaign</a></li>
+            </ul>
+         </div>
+        </nav>
```
Comment (Qoosim) [inline about.html:55]: - [ ] Kindly make both navbars aligh on the right as we have it on the guiding template.

### Your Design
your-navbar

### Guiding Template
temp-navbar

File: about.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+      <a href=""><img src="./images/premium_photo-1669559970218-ce64b5a74bea.avif" alt="" /></a>
+    </article>
+    <hr />
+
+    <section class="pastSummits">
+      <h2>See the past CC Gamers Global Summits</h2>
+      <p>
+        Take a look at the last two CC Gamers Global Summits which took place in
+        Washington DC USA
+      </p>
```
Comment (Qoosim) [inline about.html:99]: - [ ] Kindly add red underline to the heading `<h2>See the past CC Gamers Global Summits</h2>` as we have it on the guiding template.

### Your Design
your-about-underline

### Guiding Template
temp-about-underline

File: gamers.html [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        <li><a href="">About</a></li>
+        <li><a href="">Program</a></li>
+        <li><a href="">Join</a></li>
+        <li><a href="">Sponsor</a></li>
+        <li><a href="">News</a></li>
+    </ul>
+    
+    </main>
+    
+</section>
```
Comment (Qoosim) [inline gamers.html:25]: - [ ] It is expected that there should a link from from about page on mobile to home page and vice-versa. At this moment, none of the links redirects to the either of the home or about from both pages on mobile.

[mobile-menu-behave.webm](https://user-images.githubusercontent.com/26597432/233061780-0347a8f8-44a0-4a54-acf1-2f6f3b1a53a7.webm)

File: gamers.html [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+                <ul>
+                    <li><a href="about.html" class="aboutDesktop">About</a></li>
+                    <li><a href="" class="aboutLi">Program</a></li>
+                    <li><a href="" class="aboutLi">Join</a></li>
+                    <li><a href="" class="aboutLi">Sponsor</a></li>
+                    <li><a href="" class="aboutLi">News</a></li>
+                    <li ><a href="" class="border">cc campaign</a></li>
+                </ul>
+             </div>
+        </nav>
```
Comment (Qoosim) [inline gamers.html:55]: - [ ] Kindly make both navbars aligh on the right as we have it on the guiding template.

### Your Design
your-navbar

### Guiding Template
temp-navbar

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @BenjaminMatembe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @BenjaminMatembe, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good presentation :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+          </p>
+        </div>
+  
+        <div class="right">
+          <p>2023 Creative Gamers Uganda</p>
+          <span>Some rights reserved</span>
+        </div>
+      </footer>
+<div class="button">
+    <button class="more">MORE</button></div>
```
Comment (cynthiainga) [inline index.html:167]: - [OPTIONAL] I noticed. that your `MORE` button didn’t work as expected. The functionality of the `MORE` button is to show more speakers, which means when users click on the more button they can see all 6 speakers, and the `MORE` button change to `LESS`, and if the user clicks again to the button when it shows `LESS` only two speakers display and the button change again to the `MORE` 🙃 😁

**Review by cynthiainga — APPROVED** [review]
Hi @BenjaminMatembe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Correct GitHub flow ✔️ 
- [ ] Good design ✔️ 
- [ ] The live demo works correctly ✔️ 
- [ ] No linters errors ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: gamers.html (hunk 5 above)
Comment (BenjaminMatembe) [reply gamers.html]: hi @Qoosim  thank you for the advise, it has really improved my attention to details.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,223 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+<ul>
+    <li><a href="https://benjaminmatembe.github.io/gamers_conference/">Live Demo</a></li>
```
[the remark this answers is not in the document]
Comment (BenjaminMatembe) [reply README.md:80]: hi @amejid thank you for the suggestion, this has unblocked me 💯 👍 . You just put a smile on my face.

File: index.html (hunk 6 above)
Comment (BenjaminMatembe) [reply index.html:167]: hi @cynthiainga  thank you for approving my project. Am surely going to work on the optional suggestion too so I can also experience and explore that functionality.

**Pull request merged**